### ISSUE TITLE
Problem with encoding in 0.18.0

### ISSUE THREAD

**Issue by lgordey** [Rich-Harris/buble#104]
input:
```jsx
<Button>Push &lt; Me</Button>;
```
output:

**buble@v0.16.0**
```js
React.createElement( Button, null, "Push < Me" );
```

**buble@v0.18.0**
```js
React.createElement( Button, null, "Push &lt; Me" );
```

**Comment by adrianheine** [comment]
I broke this in 16675bb146b9584e4c37e264068f44d3bbab3abb.

**Comment by diegohaz** [comment]
Thanks, @adrianheine. Can we have a new release?

**Comment by adrianheine** [comment]
I'm working on it.

**Comment by adrianheine** [comment]
I just released 0.19.0.
